### PR TITLE
mobile-style: improve mobile display

### PR DESCRIPTION
**Pull request by jeffkaufman**
Rework the mobile display for the site.  The original desktop site was written around the
assumption that all screens were at least 1024px wide, so instead of fighting that, set
the viewport to 1024px and then make tweaks on top of that.  Changes, which will apply to
all screens narrower than 1024px:

* Hide the sidebar, which includes "Getting Started", "Recent Commments", "Recently on EA
  Blogs", and "Top Contributors".

* Stop indenting the body of posts, but do indent the title of posts to leave room for the
  #votes counter.

* Make various things bigger, since we're on a small screen.

This isn't perfect, but it makes it much more usable on mobile.  Primarily, the text is
now big enough to read without zooming.  Some screenshots:

  http://www.jefftk.com/eaf-article-view.png
  http://www.jefftk.com/eaf-comments-view.png
  http://www.jefftk.com/eaf-index-bottom.png
  http://www.jefftk.com/eaf-index-view.png

  (From my phone, a Galaxy S6 with weird DPI settings, so not entirely representative.)

Here's the html of an article page and the index page, with this change patched in:

  http://www.jefftk.com/concerns_with_intentional_insights
  http://www.jefftk.com/ea-index

I haven't tested this pull request except by manually patching my changes into html pages
(the two linked aboved).  That is, I haven't set up a developer environment for the EA
Forum codebase, so there might be something I'm missing about how to integrate my css
change.

### CODE REVIEW

**Comment by jeffkaufman** [comment]
Possibly this should go into `main.css` instead of `templates/base.html`, but I haven't tested that.

**Comment by patbl** [comment]
Wowowow so amaze!

Fixes https://github.com/tog22/eaforum/issues/78.

You can move the styles to `main.css`. I did that [here](https://github.com/jeffkaufman/eaforum/commit/2618eba2d0d7d4d0d8cccdb45c77ca9c893e04ec) and it seems to work. (I would also expect it to work based on my experience with the code.)

Before:
iphone5-non-responsive

After:
iphone5-responsive

Unfortunately, if someone copies and pastes a document, the font size is often included in an inline style, which overrides the style rules you added:

```
<span style="color: #333333; font-family: Arial, Helvetica, sans-serif; font-size: 13.3333px;">! Look at this mottled snowball of fuzz. Look at it! Now just try to look away. I dare you. See?&nbsp;</span>
```

iphone5-copy-paste

I've been meaning to look into cleaning up pasted input. It would be possible to override the inline styles with `!important`, but this wouldn't be great. (You couldn't just add `!important` to everything; you'd also have to make the selectors more specific.)

File: r2/r2/templates/base.html [hunk 1]
```diff
@@ -68,6 +71,64 @@
 
     ${self.head()}
     <script src="//load.sumome.com/" data-sumo-site-id="2115f2f4b9e86f1f2e73c48fb896958ad06ae47bfb80face9329b72cb21fca6c" async="async"></script>
+    <style>
+    @media only screen and (max-width: 1024px)  {
```
Comment (patbl) [inline r2/r2/templates/base.html]: What do you think about making this a bit smaller (maybe 750px)? In my opinion, the text is too big on some resized desktop windows. After (left) and before (right):

screenshot from 2016-11-27 20-47-20

Comment (jeffkaufman) [inline r2/r2/templates/base.html]: Done!

**Comment by jeffkaufman** [comment]
I ended up needing to move this to eaforum.css instead of `main.css`, because we want to override some things in `eaforum.css`.  Primarily comment font sizes.

**Comment by jeffkaufman** [comment]
Ok, I think this is ready to go in!  Any more comments?

**Comment by patbl** [comment]
Looks good to me!

**Comment by jeffkaufman** [comment]
What needs to happen before this can go live?

**Comment by peterhurford** [comment]
LGTM to me. @jeffkaufman and @patbl, is this okay to merge? If so, we can make that happen.

**Comment by jeffkaufman** [comment]
>  @jeffkaufman is this okay to merge?

 Yup!  (From my perspective.)

**Pull request merged**